about.html

```diff
@@ -4,11 +4,11 @@
 slug: about
 ---
 <p>
-LXQt is a lightweight Qt desktop environment.
+A light Qt desktop environment.
 </p>
 <p>
-It will not get in your way. It will not hang or slow down your system. It is focused
```

Comment:
This text gets in my way. It is neither focused nor modern.

Reply:
Well, I really doubt modern=better is always true. 
For me I always liked it. Taste differ.

Reply:
In the interest of keeping with "modern look and feel", _better_ is beside the point.

Reply:
As I said, taste differ, the text was there for years and in a  nutshell explained well the aims.

I forgot to ask what "focused classic" exactly is. It sounds like from a PR agency.
Some critics I read about our old about page was that there was just an assembly of buzzwords and little info.

Reply:
What it needs is screenshots and links to projects.
While it may present itself as something people can use, it is a lot better to give away that idea by letting people think for themselves given a visual representation.